moss-client/moss-client-common/src/test/java/org/xujin/moss/client/endpoint/dependency/util/PomUtilTest.java

```diff
@@ -0,0 +1,28 @@
+package org.xujin.moss.client.endpoint.dependency.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PomUtilTest {
+  @Test
+  public void testGetArtifactIdAndVersion() {
+    Assert.assertArrayEquals(
+        new String[] {
```

Comment:
I feel these could be inlined better. Can you not have the spring object on just 1 line, e.g. 
```
    Assert.assertArrayEquals(
        new String[] { "", "٢９٢٢٢٢٢٢٢٢٢٢٢٢٢٢"},
	PomUtil.getArtifactIdAndVersion("-٢９٢٢٢٢٢٢٢٢٢٢٢٢٢٢")
    );
```

Reply:
Agreed.